lib/SILOptimizer/Mandatory/TFConstExpr.cpp

```diff
@@ -1625,6 +1599,10 @@ void ConstExprEvaluator::computeConstantValues(
   unsigned numInstEvaluated = 0;
   ConstExprFunctionState state(*this, nullptr, {}, numInstEvaluated);
   for (auto v : values) {
+    LLVM_DEBUG(llvm::dbgs() << "ConstExpr: Trying to evaluate " << v
```

Comment:
heads-up that when we const-eval a lot of values (as in the case of deabstraction), this might blow up the debug output. I wish there were mechanisms for VLOG(1) vs VLOG(2). 

In its absence, perhaps we can treat LLVM_DEBUG as VLOG(1), and do not check in code for VLOG(2) (and only add such code in ad-hoc debugging sessions)?

Reply:
I'll remove it